rust/stackablectl/src/constants.rs

```diff
@@ -15,4 +15,6 @@ pub const HELM_REPO_URL_STABLE: &str = "https://repo.stackable.tech/repository/h
 pub const HELM_REPO_URL_TEST: &str = "https://repo.stackable.tech/repository/helm-test/";
 pub const HELM_REPO_URL_DEV: &str = "https://repo.stackable.tech/repository/helm-dev/";
 
-pub const CACHE_HOME_PATH: &str = "stackablectl";
+pub const XDG_APPLICATION_NAME: &str = "stackablectl";
+pub const XDG_ORGANIZATION_NAME: &str = "Stackable";
+pub const XDG_QUALIFIER: &str = "tech";
```

Comment:
Should we still prefix this stuff `XDG_*`, when xdg is only used in linux? Or maybe call it e.g. `CACHE_APPLICATION_NAME` or similar

Reply:
Renamed in 1fe6c05